warehouse/identity/identity.go

```diff
@@ -11,7 +11,7 @@ import (
 	"strings"
 	"time"
 
-	uuid "github.com/gofrs/uuid"
```

Comment:
Note: we should use google's uuid, as gofrs can cause [performance issues](https://github.com/rudderlabs/rudder-server/pull/1606)

Reply:
Thanks, @lvrach for sharing this PR.

Reply:
Although it was out of the scope of the PR, so thinking to make this change in a separate PR.

Reply:
Also, any specific reason why we haven't migrated all existing implementation everywhere from `uuid.Must(uuid.NewV4())` to `misc.FastUUID().String()`?

Reply:
Created a draft https://github.com/rudderlabs/rudder-server/pull/2598 to discuss the same.